tests/e2e/utils/fs.ts

```diff
@@ -138,6 +138,29 @@ export function expectFileToMatch(fileName: string, regEx: RegExp | string) {
     });
 }
 
+export function expectFileNotToMatch(fileName: string, regEx: RegExp | string) {
```

Comment:
@hansl I agree, I read some more tests while waiting for Travis and realized that was probably the right way to work with existing helpers.